src/components/Common/Elements/Button/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+
+interface Props {
+  children: any;
```

Comment:
```suggestion
  children: ReactNode;
```

Reply:
Giving children the `ReactNode` type will make it a bit more type safe than `any`
[An example explanation](https://egghead.io/lessons/react-use-react-reactnode-for-the-children-prop-in-react-typescript-components-and-render-props)